packages/rich-text/src/plugins/shared/LinkedInlineWrapper.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import tokens from '@contentful/f36-tokens';
+import { EntryLink, ResourceLink } from '@contentful/field-editor-reference';
+import { css } from 'emotion';
+
+import { IS_CHROME } from '../../helpers/environment';
+import { Element, RenderElementProps } from '../../internal/types';
+
+const styles = {
+  icon: css({
+    marginRight: '10px',
+  }),
+
+  root: css({
+    display: 'inline-block',
+    margin: `0 ${tokens.spacing2Xs}`,
+    fontSize: 'inherit',
+    span: {
+      userSelect: 'none',
+    },
+  }),
+};
+
+const isResourceLink = (link: EntryLink | ResourceLink): link is ResourceLink =>
```

Comment:
I think we have this one also in the LinkedBlockWrapper, can we maybe extract it to an util?
Or maybe even this snipped `isResourceLink(link) ? link.sys.urn : link.sys.id` as a function